### ISSUE TITLE
coordinate didn't get logged - now automated system that was set up won't work

### ISSUE THREAD

**Issue by dtoy** [Esri/Survey123Community#501]
The x y coordinates that the app automatically logs, didn't log. Now since I can't change that in the data that is stored in ESRI's database, my automation to get the data into MySQL won't work. I have technicians using this and I don't want them in the CSV that is downloaded. How can I get that behind the scenes location to come in? If I changed it manually on my end, I still have resaving errors that changes little pieces of data (03 now comes in as 3).

**Comment by sunyu0530** [comment]
@dtoy 

Hi, Where exactly are you trying to pull the x,y coordinates? from the Esri File geodatabase that you downloaded?

**Comment by survey123** [comment]
Hi Dustin. The best way to ensure that coordinates are captured is to add a geopoint question and flag it as required. If you  do not add a geopoint question, the app will attempt to get the XY coordinates from your device when the form is opened. Having said this, if the device does not have location services running or if the device has  it running, but for some reason the location provided by your  device is null, then the Survevy123 app will not be able to store any coordinates and your geometry will be null.

**Comment by dtoy** [comment]
ESRI automatically records an x and y location. I set up an automated system through MS Access, Python, and MySQL to get the data collected and place it into the SQL database. I assumed that the x,y coordinates would be recorded each time (I did not put this field in or require it). I also didn't have code to deal with it being blank, so my system 'broke' (I have since written some dummy code to deal with it). My concern is that the x,y coordinates can't be looked at / managed in "My Content" on arcgis.com. I just think that should be an option.
